src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -115,127 +146,98 @@ export default function BulkEmailForm(props) {
-            </strong>
-            <FormattedMessage
-              id="bulk.email.instructions.caution.message"
-              defaultMessage=" When you select Send Email, your email message is added to the queue for sending,
-              and cannot be cancelled."
-              description="A warning about how emails are sent out to users"
-            />
+            <strong>{intl.formatMessage(messages.bulkEmailInstructionsCaution)}</strong>
+            {intl.formatMessage(messages.bulkEmailInstructionsCautionMessage)}
           </p>
         </div>
-        <Form.Group className="d-flex flex-row">
-          <StatefulButton
-            variant="primary"
-            type="submit"
-            onClick={(event) => {
-              event.preventDefault();
-              openTaskAlert();
-            }}
-            state={emailFormStatus}
-            icons={{
-              default: <Icon className="icon-download" />,
-              pending: <Icon src={SpinnerSimple} className="icon-spin" />,
-              complete: <Icon src={CheckCircle} />,
-              error: <Icon src={Cancel} />,
-            }}
-            labels={{
-              default: 'Submit',
-              pending: 'Submitting',
-              complete: 'Task Created',
-              error: 'Error',
-            }}
-            disabledStates={['pending', 'complete']}
-          >
-            <FormattedMessage
-              id="bulk.email.submit.button"
-              defaultMessage="Submit"
-              description="Submit/Send email button"
+        <Form.Group>
+          <div className="mb-3">
+            <CheckBox
+              name="scheduleEmailBox"
+              checked={isScheduled}
+              onChange={() => toggleScheduled((prev) => !prev)}
+              label={intl.formatMessage(messages.bulkEmailFormScheduleBox)}
+              disabled={emailFormStatus === FORM_SUBMIT_STATES.PENDING}
+            />
+          </div>
+          {isScheduled && (
+            <ScheduleEmailForm
+              isValid={emailFormValidation.schedule}
+              onDateTimeChange={onDateTimeChange}
             />
-          </StatefulButton>
-          {emailFormStatus === FORM_SUBMIT_STATES.ERROR && (
-            <Form.Control.Feedback className="px-3" hasIcon={false} type="invalid">
-              <FormattedMessage
-                id="bulk.email.form.error"
-                defaultMessage="An error occured while attempting to send the email."
-                description="An Error message located under the submit button for the email form. Visible only on a failure."
-              />
-            </Form.Control.Feedback>
-          )}
-          {(emailFormStatus === FORM_SUBMIT_STATES.COMPLETED_DEFAULT
-            || emailFormStatus === FORM_SUBMIT_STATES.COMPLETE) && (
-            <Form.Control.Feedback className="px-3" hasIcon={false} type="valid">
-              <FormattedMessage
```

Comment:
This messaging is not in the new mocks, and so I've removed it. Theres no longer a success message when an email is successfully sent, besides the messaging on the submit button.